src/idl_gen_json_schema.cpp

```diff
@@ -164,6 +164,36 @@ class JsonSchemaGenerator : public BaseGenerator {
     return std::string(num_spaces, ' ');
   }
 
+  std::string PrepareDescription(
+      const std::vector<std::string> &comment_lines) {
+    std::string comment;
+    for (auto line_iterator = comment_lines.cbegin();
+         line_iterator != comment_lines.cend(); ++line_iterator) {
+      const auto &comment_line = *line_iterator;
+
+      // remove leading and trailing spaces from comment line
+      const auto start = std::find_if(comment_line.begin(), comment_line.end(),
+                                      [](char c) { return !isspace(c); });
+      const auto end = std::find_if(comment_line.rbegin(), comment_line.rend(),
+                                    [](char c) { return !isspace(c); })
+                           .base();
+      std::string trimmed_line(start < end ? std::string(start, end)
```

Comment:
No need to make a copy first, can simply pass iterators directly to `append`.

Reply:
True, this is not needed. I changed this. May I ask another question regarding clang-format? I have the issue that I formatted the code with clang-format, but this will fail in the CI. Is there a special version I have to use?

Reply:
Not sure.. we can just submit it if that's the only problem.